tools/pipelines/test-real-service.yml

```diff
@@ -60,7 +60,7 @@ stages:
         testWorkspace: ${{ variables.testWorkspace }}
         artifactBuildId: $(resources.pipeline.client.runID)
         testCommand: test:realsvc:tinylicious:report:full
-        timeoutInMinutes: 90
+        timeoutInMinutes: 120
```

Comment:
Should we leave a TODO comment here that this should be reduced back to 90 once we've resolved the issue? Probably worth referencing the appropriate work item as well.